weblate/templates/translate.html

```diff
@@ -35,65 +35,66 @@
 {% endif %}
 
 <a href="{% url 'profile' %}#preferences" class="btn btn-link" title="{% trans "Configure editor" %}">{% icon "settings.svg" %}</a>
 </div>
-
-{% with comments=unit.get_comments nearby=unit.nearby nearby_keys=unit.nearby_keys shapings=unit.shapings%}
-
-<div class="btn-group">
-<ul class="pagination unit-navi">
-<li><a id="button-first" class="green" href="{{ first_unit_url }}" title="{% trans "First" %}">{% if LANGUAGE_BIDI %}{% icon "page-last.svg" %}{% else %}{% icon "page-first.svg" %}{% endif %}</a></li>
-<li><a id="button-prev" class="green" href="{{ prev_unit_url }}" title="{% trans "Previous" %}">{% if LANGUAGE_BIDI %}{% icon "page-next.svg" %}{% else %}{% icon "page-previous.svg" %}{% endif %}</a></li>
-<li class="dropdown">
-<a class="dropdown-toggle green" data-toggle="dropdown" href="#" title="{% trans "Edit search parameters" %}" id="search-dropdown">
-{% icon "magnify.svg" %}
-{{ filter_name }}
-<span class="caret"></span>
-</a>
-  <div class="dropdown-menu">
-
-<div class="panel-group">
-<form action="{{ unit.translation.get_translate_url }}" method="GET">
-{{ search_form|crispy }}
-<input type="submit" value="{% trans "Search" %}" class="btn btn-primary" />
-</form>
 </div>
 
-</div>
-</li>
-<li class="dropdown">
-<a class="dropdown-toggle" data-toggle="dropdown" href="#" title="{% trans "Go to position" %}" id="goto-dropdown">
-{% blocktrans %}{{ filter_pos }} / {{ filter_count }}{% endblocktrans %}
-<span class="caret"></span>
-</a>
-  <div class="dropdown-menu">
-
-<div class="panel-group">
-
-<form action="{{ unit.translation.get_translate_url }}" method="GET">
-{% for key, value in search_items %}
-<input type="hidden" name="{{ key }}" value="{{ value }}" />
-{% endfor %}
-<div class="form-group">
-<label class="control-label" for="id-goto-number">{% trans "Jump to position" %}</label>
-<div class="controls">
-<input type="number" min="1" max="{{ filter_count }}" value="{{ filter_pos }}" name="offset" id="id-goto-number">
-</div>
-</div>
-<input type="submit" value="{% trans "Jump" %}" class="btn btn-primary" />
-</form>
-</div>
+{% with comments=unit.get_comments nearby=unit.nearby nearby_keys=unit.nearby_keys shapings=unit.shapings%}
 
-</div>
-</li>
-<li><a id="button-next" class="green" href="{{ next_unit_url }}" title="{% trans "Next" %}">{% if not LANGUAGE_BIDI %}{% icon "page-next.svg" %}{% else %}{% icon "page-previous.svg" %}{% endif %}</a></li>
-<li><a id="button-end" class="green" href="{{ last_unit_url }}" title="{% trans "Last" %}">{% if not LANGUAGE_BIDI %}{% icon "page-last.svg" %}{% else %}{% icon "page-first.svg" %}{% endif %}</a></li>
-</ul>
+<div class="row">
+  <div class="col-sm-9">
+    <div class="panel-group">
+      <form action="{{ unit.translation.get_translate_url }}" method="GET">
+        {% include "snippets/query-field.html" with field=search_form.q  %}
+      </form>
+    </div>
+  </div>
+  <div class="col-sm-3">
+    <div class="btn-group">
```

Comment:
```suggestion
    <div class="btn-group pull-right flip">
```

How about right aligning this? But it is probably not that much better.

Reply:
I will check how it looks UI wise. I just don't wanna make it seem un-intuitive too much to relate that the pagination is related to the translation.

Reply:
Let's skip UI twiddling for now, I will get proper UI design for this part.